caffe2/perfkernels/embedding_lookup_idx_avx2.cc

```diff
@@ -5,9 +5,10 @@
 //// DO NOT MODIFY!!!
```

Comment:
See comment here?

Reply:
While the file was generated from python, it has been modified several times, as you can see from the git history..